docs/json_schemas/grafana_datasource_exchange/v0/provider.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$defs": {
+    "BaseModel": {
+      "properties": {},
+      "title": "BaseModel",
+      "type": "object"
+    },
+    "GrafanaDatasource": {
+      "properties": {
+        "type": {
+          "description": "Type of the datasource.",
+          "examples": [
+            "tempo",
```

Comment:
I wonder if these shouldn't be `traces`, `logs`, `metrics` not to focus on a specific tool we use. That way it also doesn't matter if it's prometheus or mimir. However this won't change the type itself.

Reply:
so, it's not an arbitrary string but a datasource type name as defined by grafana. You can define custom ones but users don't have the freedom to just put whatever there. Will add a link to the upstream docs on the matter.